hedy.py

```diff
@@ -1249,6 +1250,7 @@ def equality_check(self, args):
     def in_list_check(self, args):
         arg0 = process_variable(args[0], self.lookup)
         arg1 = process_variable(args[1], self.lookup)
+        self.check_var_usage([arg1])
```

Comment:
For this case, it would have been really nice to see a test added that catches an error that we did not catch before. Can you add one? Something like:

```if dier in dieren:
...```

Used to work, but now will now give a (desired!) exception.

Also shouldn't we tests both sides here?

Reply:
Looked into it and this one is not needed, type checker can handle this